doc/en/user/source/installation/docker.rst

```diff
@@ -163,7 +163,6 @@ Here is a list of available extensions (taken from the `build server <https://bu
     geofence-wps             ogcapi-features          wps                      
     geopkg-output            ogr-wfs                  ysld                     
 
-
```

Comment:
imagemap was left amongts the community modules

Reply:
I've also found a reference to imagemap in an odd place:

src/extension/importer/core/src/test/java/org/geoserver/importer/ModuleStatusTest.java:     
* public void test() { assertModuleStatus("gs-imagemap", "ImageMap Extension"); }

It's commented out code actually... LOL weird. Feel free to ignore, I was just checking with "git grep".